model/event_models.go

```diff
@@ -105,3 +105,14 @@ type SourceEvent struct {
 	SourceTypeID        *int64  `json:"source_type_id"`
 	Tenant              *string `json:"tenant"`
 }
+
+type RhcConnectionEvent struct {
+	RhcId *string        `json:"uuid"`
```

Comment:
We're doing to return the field as `rhc_id` instead of `uuid`.

Reply:
Changed. I also included the `ID` field that was missing.